### ISSUE TITLE
Reconplogger statements are sometimes replaced by plain_logger.

### ISSUE THREAD

**Issue by didirus** [omni-us/reconplogger#12]
I have reconplogger 4.7.0 installed.

When I run my code, it sometimes logs my logger statements with `plain_logger` instead of `reconplogger` and uses logger level `DEBUG` instead of the configured logger level for reconplogger which was set to `INFO`. 

Downgrading to reconplogger 4.3.1 worked to solve this problem.

See example of my logs:

```2021-03-18 11:26:26,691    INFO -- training.py:563 -- Size val: 3
2021-03-18 11:26:26,691    INFO -- training.py:564 -- Number of classes found: 2
2021-03-18 11:26:29,633    INFO -- training.py:464 -- Weighted loss: Set manual weights in '--training.weighted_loss.weights' by copying+editing this list: [1.5, 0.71]
2021-03-18 11:26:29,643    INFO -- training.py:120 -- Selecting images for gradcam visualisation...
2021-03-18 11:26:29,945    INFO -- training.py:123 -- Groundtruth given: True
2021-03-18 11:26:29,946    INFO -- training.py:130 -- ========================================
2021-03-18 11:26:29,946    INFO -- training.py:143 -- Training time limit: 4.0 seconds
2021-03-18 11:26:29,946    INFO -- training.py:144 -- ========================================
DEBUG:plain_logger:Confusion matrix:
   C  M
C  0  1
M  0  2
2021-03-18 11:26:30,586    INFO -- training.py:234 -- Epoch    0/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 0.81
INFO:plain_logger:Epoch    0/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 0.81
2021-03-18 11:26:32,495    INFO -- training.py:234 -- Epoch    1/30 train_acc: 0.50, train_ma-f1: 0.45 train_loss: 1.16, lr: 0.01
INFO:plain_logger:Epoch    1/30 train_acc: 0.50, train_ma-f1: 0.45 train_loss: 1.16, lr: 0.01
DEBUG:plain_logger:Confusion matrix:
   C  M
C  0  1
M  0  2
2021-03-18 11:26:33,007    INFO -- training.py:234 -- Epoch    1/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 4.36
INFO:plain_logger:Epoch    1/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 4.36
2021-03-18 11:26:34,851    INFO -- training.py:234 -- Epoch    2/30 train_acc: 0.60, train_ma-f1: 0.58 train_loss: 2.28, lr: 0.01
INFO:plain_logger:Epoch    2/30 train_acc: 0.60, train_ma-f1: 0.58 train_loss: 2.28, lr: 0.01
DEBUG:plain_logger:Confusion matrix:
   C  M
C  0  1
M  0  2
2021-03-18 11:26:35,353    INFO -- training.py:234 -- Epoch    2/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 2.24
INFO:plain_logger:Epoch    2/30   val_acc: 0.67,   val_ma-f1: 0.40   val_loss: 2.24
2021-03-18 11:26:35,353    INFO -- training.py:238 -- Stopping training, because time limit is reached.
INFO:plain_logger:Stopping training, because time limit is reached.
2021-03-18 11:26:35,359    INFO -- training.py:243 -- Training complete in 0.000000m 6s
INFO:plain_logger:Training complete in 0.000000m 6s
2021-03-18 11:26:35,598    INFO -- evaluation.py:59 -- ========================================
INFO:plain_logger:========================================
2021-03-18 11:26:35,598    INFO -- evaluation.py:60 -- Results of best model on val set
INFO:plain_logger:Results of best model on val set
2021-03-18 11:26:35,598    INFO -- evaluation.py:61 -- ========================================
INFO:plain_logger:========================================
2021-03-18 11:26:38,419    INFO -- evaluation.py:179 -- Accuracy = 0.67
INFO:plain_logger:Accuracy = 0.67
2021-03-18 11:26:40,635    INFO -- training.py:297 -- Gradcam results are saved in ...
INFO:plain_logger:Gradcam results are saved in...
2021-03-18 11:26:40,637    INFO -- training.py:327 -- Best model found after epoch 0. ma-F1: 0.40 (val-set).
INFO:plain_logger:Best model found after epoch 0. ma-F1: 0.40 (val-set).
2021-03-18 11:26:40,637    INFO -- training.py:328 -- For training summaries, run "..."
INFO:plain_logger:For training summaries, run "tensorboard --logdir ..."
2021-03-18 11:26:40,653    INFO -- training.py:72 -- Evaluation results are saved in ...
INFO:plain_logger:Evaluation results are saved ...
```

**Comment by mauvilsa** [comment]
This is a known issue which currently don't know what causes it, see https://github.com/omni-us/reconplogger#known-issues. This problem was identified long ago and was fixed using a kind of hack. But this hack caused other problems so we removed it. You now have to resolve this in your code. Basically look at where in your code the logging package is used as a logger and remove it or change it to use reconplogger.
